protected/views/ethnicGroup/view.php

```diff
@@ -0,0 +1,86 @@
+<?php
+/* @var $this EthnicGroupController */
+/* @var $model EthnicGroup */
+$this->pageTitle = 'View Ethnic';
```

Comment:
Any reference to Ethnic should probably be Ethnic Group or Ethnicity. Either of these are fine but whatever term you choose to use should be reflected throughout the new screens.